Resources.it.resx

```diff
@@ -0,0 +1,1044 @@
+    <comment>Settings bandwidth out alert tooltip</comment>
+  </data>
+  <data name="SettingsClickThrough" xml:space="preserve">
+    <value>Clicca attraverso</value>
+    <comment>Settings click through label</comment>
+  </data>
+  <data name="SettingsClickThroughTooltip" xml:space="preserve">
+    <value>Fa passare gli eventi del mouse attraverso la finestra. Nota che ciò impedirà qualsiasi interazione con la barra laterale.</value>
+    <comment>Settings click through tooltip</comment>
+  </data>
+  <data name="SettingsCollapseMenuBar" xml:space="preserve">
+    <value>Comprimi barra dei menu</value>
+    <comment>Settings collapse menu bar label</comment>
+  </data>
+  <data name="SettingsCollapseMenuBarTooltip" xml:space="preserve">
+    <value>Rimuove lo spazio aggiuntivo nella parte superiore della barra laterale.</value>
+    <comment>Settings collapse menu bar tooltip</comment>
+  </data>
+  <data name="SettingsCoreLoads" xml:space="preserve">
+    <value>Mostra carichi principali</value>
+    <comment>Settings core loads label</comment>
+  </data>
+  <data name="SettingsCoreLoadsTooltip" xml:space="preserve">
+    <value>Mostra il carico percentuale di tutti i core.</value>
+    <comment>Settings core loads tooltip</comment>
+  </data>
+  <data name="SettingsCustomizeTab" xml:space="preserve">
+    <value>Personalizza</value>
+    <comment>Title of the settings customize tab</comment>
+  </data>
+  <data name="SettingsDateFormat" xml:space="preserve">
+    <value>Formato data</value>
+    <comment>Settings date format label</comment>
+  </data>
+  <data name="SettingsDateFormatDisabled" xml:space="preserve">
+    <value>Disabilitato</value>
+    <comment>Settings date format disabled value</comment>
+  </data>
+  <data name="SettingsDateFormatTooltip" xml:space="preserve">
+    <value>Formato della data sotto l'orologio.</value>
+    <comment>Settings date format tooltip</comment>
+  </data>
+  <data name="SettingsDock" xml:space="preserve">
+    <value>Aggancia</value>
+    <comment>Dock settings label</comment>
+  </data>
+  <data name="SettingsDockBottom" xml:space="preserve">
+    <value>Parte inferiore</value>
+    <comment>Dock bottom settings label</comment>
+  </data>
+  <data name="SettingsDockLeft" xml:space="preserve">
+    <value>Sinistra</value>
+    <comment>Dock left settings label</comment>
+  </data>
+  <data name="SettingsDockRight" xml:space="preserve">
+    <value>Destra</value>
+    <comment>Dock right settings label</comment>
+  </data>
+  <data name="SettingsDockTooltip" xml:space="preserve">
+    <value>A quale bordo verrà agganciata la barra laterale.</value>
+    <comment>Dock settings tooltip</comment>
+  </data>
+  <data name="SettingsDockTop" xml:space="preserve">
+    <value>Superiore</value>
+    <comment>Dock top settings label</comment>
+  </data>
+  <data name="SettingsDriveDetailsTooltip" xml:space="preserve">
+    <value>Mostra i dettagli dell'unità extra come testo.</value>

```

Comment:
impostazioni dettagli dell'unità aggiuntiva

Reply:
Maybe this is better? => Mostra i dettagli dell'unità aggiuntiva
I would remove the 'as a text' (come testo). Also, I use 'unita' aggiuntiva' for additional drive, which I think has been used in other labels.

Reply:
OK

Reply:
@Pols88 please update the PR to use value: `Mostra i dettagli dell'unità aggiuntiva`
Thank you